website/content/docs/concepts/jobs.mdx

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+page_title: Jobs
+description: |-
+  Learn about Nomad's Job feature, which is how you deploy your apps.
+---
+
+# Jobs
+
+This page explains the concept of a job, which is the Nomad construct for
+deploying your app and its configuration.
+
+
+## Background
+
+A job is a declarative specification of tasks that Nomad should run. Each job
+can have one or more task groups, which are collections of one or more tasks.
+
+Declare the the desired state of your job in a job specification that describes
+the type of job, the tasks and resources necessary for the job to run, and job
+information such as which clients it can run on. Refer to the [Job
+Specification](/nomad/docs/job-specification) for complete details and examples.
+
+When you submit your job specification, Nomad creates a job along with
+[allocations](/nomad/docs/glossary#allocation) for each group defined in the
+spec. Nomad makes sure the actual state matches your desired state.
+
+## Job types
+
+Nomad supports the following job types:
+
+- Batch
+
+  Batch jobs are for short lived jobs that run until they exit
+  successfully. Batch jobs are paramaterized or periodic.
+    - Paramaterized: The paramaterized type accepts required or optional inputs. Trigger this job with the `job dispatch` [command](/nomad/docs/commands/job/dispatch).
+    - Periodic: The periodic type is a Nomad cron job that runs at configured times.
+
+- Service
+
+  Service jobs are for long lived services that run until explicitly
+  stopped. Use a service job type for your app and its resources.
+
+- Sysbatch
+
+  Sysbatch jobs are for run-once commands that should run on every node in the
+  cluster. These jobs run until explicitly stopped by an operator, evicted
+  through preemption, or successful completion.
+
+- System
+
+  System jobs run on all clients that meet the job's constraints. Use a system
+  job for deploying and managing tasks that should be present on every node in
+  the cluster.
+
+Refer to the [Schedulers](/nomad/docs/schedulers) page for details on how Nomad schedules each of these job types.
+
+## Job statuses
+
+After you submit your job, Nomad assigns a status to indicate how a job is
+performing. This status is based on running, healthy [allocations](/nomad/docs/glossary#allocation) versus desired job state.
+
+All jobs may have the following statuses in the Nomad UI:
+
+- Degraded: A deployment is not taking place. Some allocations are failed, lost, or unplaced.
+- Deploying: A [deployment](/nomad/docs/glossary#deployment) is actively taking place.
+- Failed: All allocations are failed, lost, or unplaced.
+- Healthy: All expected allocations are running and healthy.
+- Recovering: Some allocations are pending.
+- Removed: The job appeared in the initial query but has since been garbage
+  collected. This happens when the user remains on the page for a long time.
+- Stopped: A user has has manually stopped the job. It has not yet been purged or garbage collected.
+
```

Comment:
We've also added `Scaled Down` as a status as of https://github.com/hashicorp/nomad/pull/23829 / https://github.com/hashicorp/nomad/issues/23591 (next 1.8.x release)

Reply:
I decided to add this status. I looked at the linked PR/issue to create the definition and its placement in the status mapping table.